addons/analytic_user_function/analytic_user_function.py

```diff
@@ -116,8 +116,8 @@ def on_change_account_id(self, cr, uid, ids, account_id, user_id=False, unit_amo
             res ['value']['general_account_id']= a
         return res
 
-    def on_change_user_id(self, cr, uid, ids, user_id, account_id, unit_amount=0):
-        res = super(hr_analytic_timesheet, self).on_change_user_id(cr, uid, ids, user_id)
+    def on_change_user_id(self, cr, uid, ids, user_id, is_timesheet, context, unit_amount=0, account_id=None):
```

Comment:
I would limit the potential issues by keeping the same order of arguments: cr, uid, ids, user_id, account_id=None, unit_amount=0, is_timesheet=False, context=None)